microshift_running_apps/microshift-deleting-resource-manifests.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-deleting-resource-manifests"]
+= Deleting resource manifests
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-deleting-resource-manifests
+
+toc::[]
+
+{microshift-short} supports the deletion of resource manifests for the following use cases:
+
+* Data removal - when you need to completely remove a resource from the cluster.
+* Upgrade - during an upgrade, some resources may need to be removed while others are retained to preserve data.
+
+Deleting resource manifests manages the cluster resources efficiently by removing outdated objects while and retaining necessary data.
```

Comment:
This sounds like this is used for stale object, but if someone would create new manifest, old object would live side by side - in case of Pods/Deployments/etc. they would still run potentially making problems

Reply:
done